client/src/components/Shared/CircleImg/CircleImg.jsx

```diff
@@ -6,7 +6,12 @@ import { Wrapper, Subtitle, Image, PlusCircle } from './CircleImg.styled';
 const CircleImg = ({ type, size, imgUrl, subtitle, clickHandler }) => (
 	<Wrapper>
 		{type !== 'wPlus' ? (
-			<Image type={type} size={size} onClick={clickHandler} src={imgUrl} />
+			<Image
+				type={type}
+				size={size}
+				onClick={clickHandler}
+				src={type === 'avatar' ? `https://${imgUrl}` : imgUrl}
```

Comment:
hmm, I've checked the usage of this component and nothing showed up. I remember faintly that you've changed something somewhere regarding missing protocols for Gravatars but thought that it was only for the profile pic in the navigation bar. Anyway, gonna check this again and fix if needed.

Reply:
I changed the default Gravatar profile in `authRoutes` (when registering) to add `https://` by default so that it didn't need to be added everywhere it's used (currently it's on line 38 of `authRoutes`). So it is only being added for users that register _after_ that change.